clouddriver-consul/src/main/groovy/com/netflix/spinnaker/clouddriver/consul/api/v1/model/ServiceResult.groovy

```diff
@@ -19,18 +19,18 @@ package com.netflix.spinnaker.clouddriver.consul.api.v1.model
 import com.fasterxml.jackson.annotation.JsonProperty
 
 class ServiceResult {
-  @JsonProperty("ID")
-  String iD
+  @JsonProperty("id")
```

Comment:
Please drop a comment here explaining why this combination of `@JsonProperty`and field naming is required.